assets/js/components/MenuComponent.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div>
+    <div id="viewport">
+      <!-- Sidebar -->
+      <div id="sidebar">
+        <header>
+          <a href="#">My App</a>
+        </header>
+        <ul class="nav">
+          <li>
+            <a href="#">
+              <i class="zmdi zmdi-view-dashboard"></i> Dashboard
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="zmdi zmdi-link"></i> Shortcuts
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="zmdi zmdi-widgets"></i> Overview
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="zmdi zmdi-calendar"></i> Events
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="zmdi zmdi-info-outline"></i> About
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="zmdi zmdi-settings"></i> Services
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="zmdi zmdi-comment-more"></i> Contact
+            </a>
+          </li>
+        </ul>
+      </div>
+      <!-- Content -->
+      <div id="content">
+        <nav class="navbar navbar-default">
+          <div class="container-fluid">
+            <ul class="nav navbar-nav navbar-right">
+              <li>
+                <a href="#"><i class="zmdi zmdi-notifications text-danger"></i>
+                </a>
+              </li>
+              <li><a href="#">Test User</a></li>
+            </ul>
+          </div>
+        </nav>
+        <div class="container-fluid">
+          <h1>Simple Sidebar</h1>
+          <p>
+            Make sure to keep all page content within the
+            <code>#content</code>.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
```

Comment:
Just asking, if CSS should be contained in the same file as html ?

Reply:
It is vue js component. Yes. This is how it is suppose to be.